themes/classic/assets/css/error_rtl.css

```diff
@@ -0,0 +1,50 @@
+#body {
```

Comment:
I assume this file is generated and the weird indentation is normal ?

Reply:
Absolutely, it was generated from this : https://devdocs.prestashop.com/1.7/themes/reference/rtl/#stylesheet-generation